src/appimagetool.c

```diff
@@ -611,7 +611,7 @@ main (int argc, char *argv[])
     }
 
     fprintf(
-        stderr,
+        stdout,
```

Comment:
I think we need to alternate between stdout and stderr. Only if `showVersionOnly` is set we want to display it on stdout. You can do this inline here:

```suggestion
        showVersionOnly ? stdout : stderr,
```